src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/__tests__/SAP_Internal_Stats_LinkCard+Mock.swift

```diff
@@ -2,10 +2,8 @@
 // 🦠 Corona-Warn-App
 //
 
-@testable import ENA
-
 extension SAP_Internal_Stats_LinkCard {
-	static let rkiPandemicRadarURL: String = "https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Situationsberichte/COVID-19-Trends/COVID-19-Trends.html?__blob=publicationFile#/home"
+	static let rkiPandemicRadarURLMock: String = "https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Situationsberichte/COVID-19-Trends/COVID-19-Trends.html?__blob=publicationFile#/home"
```

Comment:
by mistake?

Reply:
The reason I changed it to mock is, that we only use that for mock purposes. But it's the global extension of `SAP_Internal_Stats_LinkCard`. Of course, no part of that Task, but drops a Swift Lint warning, I saw yesterday that comes with a usage from Puneet in the ENA Target.